bagua/torch_api/distributed.py

```diff
@@ -29,7 +29,8 @@
 from .algorithms.decentralize import DecentralizedReducer
 from .algorithms.allreduce import Allreducer, ScatterGatherAllreducer
 from .compression import Compressor
-from bagua.service.autotune_service import BaguaHyperparameter
+import traceback
```

Comment:
does not seem to be used

Reply:
Used for torch hook debugging information